### PR TITLE
Remove word

### PR DESCRIPTION
**Pull request by okozmovskaya**
related #3 

Guys, I added script but I'm not sure that it works correct because the button add now isn't work. I'm ready to fix.

### CODE REVIEW

File: scripts/user-actions/remove.js [hunk 1]
```diff
@@ -3,24 +3,24 @@ console.log('--- loading: remove.js');
...
     return;
   }
 
   const isUserInput = (entry) => {
-    return _;
+    return entry === userInput;
   };
   const inputIsInArray = words
-    ._(_);
+    .find(isUserInput);
```
Comment (Sayed94h) [inline scripts/user-actions/remove.js:15]: Hi @okozmovskaya Olga, I tested your codes and it worked here but just one small thing: it has no problem but is good that line 14 and 15 are in one line, like this: 
```js 
const inputIsInArray = words.find(inUserInput); 
``` 
But if you dont change is it , it is still ok and it works!

Greetings, Sayed

**Review by Sayed94h — COMMENTED** [review]
Hi @okozmovskaya  I left you a comment at line number 14. 

Your code works. Nice work!
I am not going to merge the PR because the other reviewers will also review your PR.

Greetings, Sayed

File: scripts/user-actions/remove.js (hunk 1 above)
Comment (okozmovskaya) [reply scripts/user-actions/remove.js:15]: Thank you, Sayed for your checking and advise!

**Comment by oguzkarademir** [comment]
> but I'm not su

@okozmovskaya your code is working, good work 👍 . `display.js` file is enough to check if it is correct, after you wrote your code you can check on live server

**Review by oguzkarademir — APPROVED** [review]

**Pull request merged**